src/libnvme.map

```diff
@@ -7,7 +7,6 @@ LIBNVME_1.0 {
 		nvme_attach_ns;
 		nvme_compare;
 		nvme_copy;
-		nvme_create_ctrl;
```

Comment:
Obviously, we need also the corresponding patch to nvme-cli

Reply:
Well, this breaks building nvme-cli using the shared library. I think we should really align such changes.

Reply:
Sadly I haven't figured out how to mark a function as 'not present' in newer shared library versions.
It's easy to add new functions, but one cannot easily remove them...

Reply:
The only idea I have is to list all available symbols all the time, e.g. 

LIBVNME_1.0 {
  nvme_foo;
  nvme_bar;
}

LIBNVME_1.1 {
  nvme_foo;
}